simq/src/main/java/com/simprints/simq/SimQ.kt

```diff
@@ -0,0 +1,122 @@
+package com.simprints.simq
+
+import android.graphics.Bitmap
+import com.simprints.simq.analysis.AlignmentAnalysis
+import com.simprints.simq.analysis.BlurAnalysis
+import com.simprints.simq.analysis.BrightnessAnalysis
+import com.simprints.simq.analysis.ContrastAnalysis
+import com.simprints.simq.utils.OpenCVLoader
+import com.simprints.simq.utils.QualityUtils
+
+object SimQ {
```

Comment:
Libraries with loaded state/resources should generally avoid being objects because it forces the loading to happen on the application start (classpath load), which affects the startup times and could even lead to resource loading on the main thread. 

This should be a regular class, and leave the instance creation in the caller's control.